### PR TITLE
feat: add modal in new project

### PR DESCRIPTION
**Pull request by vwaizer**

### CODE REVIEW

**Review by ng-ngoc-hung — APPROVED** [review]

**Review by ng-ngoc-hung — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
```
Comment (dlnanh85) [inline package.json:13]: We are using TailwindCSS utility classes for styling. It's best not to introduce more CSS implementation in the project.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,9 @@
...
+    "@emotion/styled": "^11.11.5",
```
Comment (dlnanh85) [inline package.json:15]: We are using TailwindCSS utility classes for styling. It's best not to introduce more CSS implementation in the project.

File: src/components/Header/index.tsx [hunk 3]
```diff
@@ -1,16 +1,70 @@
...
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+};
```
Comment (dlnanh85) [inline src/components/Header/index.tsx:20]: Use TailwindCSS classes.

File: src/components/Header/index.tsx [hunk 4]
```diff
@@ -1,16 +1,70 @@
...
 
 export default function Header() {
+  const [isOpenModal, setIsOpenModal] = useState(false);
+  const handleOpen = () => setIsOpenModal(true);
+  const handleClose = () => setIsOpenModal(false);
```
Comment (dlnanh85) [inline src/components/Header/index.tsx:25]: For short event handlers (1-2 lines), it is common to put them right at the JSX, as an annonymous arrow function.

File: src/components/SideBar/index.tsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import Image from 'next/image';
 
 export default function SideBar() {
   return (
-    <div className='h-full w-[15%] border-r-[1px] border-[#ccc] px-2 py-4'>
+    <div className='h-full w-[15%] cursor-pointer border-r-[1px] border-[#ccc] px-2 py-4'>
```
Comment (dlnanh85) [inline src/components/SideBar/index.tsx:5]: The hover styling should not apply to the whole the sidebar `div`, but its items.

**Review by dlnanh85 — CHANGES_REQUESTED** [review]

**Review by ng-ngoc-hung — DISMISSED** [review]

**Pull request closed without merging**